src/Host/Client/Test/ExecutionTracing/BreakpointsTest.cs

```diff
@@ -18,6 +18,8 @@
 
 namespace Microsoft.R.ExecutionTracing.Test {
     [ExcludeFromCodeCoverage]
+    [Category.R.ExecutionTracing]
+    [Collection(CollectionNames.NonParallel)]
```

Comment:
OK, it was attempt to make debugger breakpoints setting tests pass but this they still fail, i can put it back.